<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Achu" failures="2" tests="22" name="Test" time="214.332" errors="0" timestamp="2023-10-20T11:56:19 BST">
  <testcase classname="testCases.CaAddAccountTest" name="createAccount" time="10.561"/>
  <testcase classname="testCases.CaAddAccountTest" name="createAccExistEmail" time="22.98"/>
  <testcase classname="testCases.CaAddAccountTest" name="createAccInvalid" time="9.525"/>
  <testcase classname="testCases.CaAddAccountTest" name="createAccEmpty" time="9.604"/>
  <testcase classname="testCases.CaAddAccountTest" name="createAccMandatoryFields" time="11.389"/>
  <testcase classname="testCases.CaLoginTest" name="LoginFormValid" time="9.286"/>
  <testcase classname="testCases.CaLoginTest" name="InvalidloginInfo" time="4.794"/>
  <testcase classname="testCases.CaLoginTest" name="emptylogin" time="3.598"/>
  <testcase classname="testCases.CaLoginTest" name="ValiduserInvalidpass" time="4.775"/>
  <testcase classname="testCases.CaLoginTest" name="Invaliduservalidpass" time="4.459"/>
  <testcase classname="testCases.CaLoginTest" name="InactiveUsernameAndPassowrd" time="4.462"/>
  <testcase classname="testCases.CaLoginTest" name="loginAndBackTest" time="6.497">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	URL Mismatch expected [https://capital-analytical.myshopify.com/account/login] but found [https://capital-analytical.myshopify.com/account?analytics_trace_id=c62018d8-5041-4e41-91d8-99dcfeb15143]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
URL Mismatch expected [https://capital-analytical.myshopify.com/account/login] but found [https://capital-analytical.myshopify.com/account?analytics_trace_id=c62018d8-5041-4e41-91d8-99dcfeb15143]
at pages.CaLoginPage.loginAndBack(CaLoginPage.java:176)
at testCases.CaLoginTest.loginAndBackTest(CaLoginTest.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginAndBackTest -->
  <testcase classname="testCases.CaAddressTest" name="Login" time="5.189"/>
  <testcase classname="testCases.CaAddressTest" name="AddressPage" time="5.548"/>
  <testcase classname="testCases.CaAddressTest" name="AddEmptyAddress" time="4.231"/>
  <testcase classname="testCases.CaAddressTest" name="CancelButton" time="5.518">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Message Mismatch expected [] but found [Add a new address]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Message Mismatch expected [] but found [Add a new address]
at pages.CaAddressPage.clickCancelBtn(CaAddressPage.java:137)
at testCases.CaAddressTest.CancelButton(CaAddressTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- CancelButton -->
  <testcase classname="testCases.CaAddressTest" name="EditAddress" time="8.725"/>
  <testcase classname="testCases.CaResetPasswordTest" name="restPasswordCorrectDetail" time="8.069"/>
  <testcase classname="testCases.CaResetPasswordTest" name="resetPasswordEmptyField" time="3.111"/>
  <testcase classname="testCases.CaResetPasswordTest" name="resetPasswordInvalid" time="5.167"/>
  <testcase classname="testCases.CaResetPasswordTest" name="resetPasswordCancelLink" time="4.026"/>
  <testcase classname="testCases.CaResetPasswordTest" name="resetPassInvalidEmailFor" time="1.968"/>
</testsuite> <!-- Test -->
